<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneControllerWindow</class>
 <widget class="QWidget" name="SceneControllerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>359</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>614</width>
    <height>359</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_measureDistance">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>206</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>距离测量</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_pointA">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>点A</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_pointAValue">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>( 0.00, 0.00, 0.00)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_pointB">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>点B</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_pointBValue">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>( 0.00, 0.00, 0.00)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_distanceAB">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>距离(米)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_distanceABValue">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>长度</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_showButtons">
       <property name="maximumSize">
        <size>
         <width>206</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>显示内容</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_showModel">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>三维模型</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_showCoverBox">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>包围盒</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_showOctreeBoxs">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>八叉树块</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_showRoamPath">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>漫游轨迹</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="pushButton_showCoordinate">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>坐标轴</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Empty">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_changeCoverBox">
       <property name="maximumSize">
        <size>
         <width>206</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>包围盒尺寸</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_sliderX">
            <property name="minimumSize">
             <size>
              <width>21</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>21</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>x轴</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderX1">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderX2">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="value">
             <number>99</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_sliderY">
            <property name="minimumSize">
             <size>
              <width>21</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>21</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>y轴</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderY1">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderY2">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="value">
             <number>99</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_silderZ">
            <property name="minimumSize">
             <size>
              <width>21</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>21</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>z轴</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderZ1">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderZ2">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="value">
             <number>99</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="AirPhoto" name="widget_AirPhoto" native="true">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10000</width>
           <height>300</height>
          </size>
         </property>
         <property name="text" stdset="0">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_roamClear">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>重置漫游点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_roamStart">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>漫游</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_currentPosition">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>当前坐标</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_currentPositionValue">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>( 0.00, 0.00, 0.00)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AirPhoto</class>
   <extends>QWidget</extends>
   <header>airphoto.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
